core/commands/location/create/create_location_command_handler.py

```diff
@@ -5,28 +5,19 @@
 from core.data_access.repositories.location_repository import LocationRepository
 from core.common.mediator import RequestHandler
 from core.dtos.location_dto import LocationDto
-from core.serializers.location.create_location_cmd_serializer import CreateLocationCommandSerializer
 
 
 class CreateLocationCommandHandler(RequestHandler[CreateLocationCommand, Result[LocationDto]]):
     def __init__(self):
         self.location_repository = LocationRepository(Location)
 
     def handle(self, command: CreateLocationCommand) -> Result[LocationDto]:
-        serializer = CreateLocationCommandSerializer(data={
-            'locationname': command.locationname,
-            'address': command.address,
-            'city': command.city
-        })
-        if not serializer.is_valid():
-            return Result.fail(error=serializer.errors, status_code=400)
+        location = Location(None, command.locationname, command.address, command.city) 
 
-        location_data = serializer.validated_data
+        if self.location_repository.location_exists(location_name=location.locationname):
```

Comment:
i think key "location_name" should be "locationname", because the method "location_exists" takes **kwargs, meaning it just inputs the key=val directly into filter, and the table column it looks up in is called "locationname".

Reply:
will test